### PR TITLE
36 create comments component

### PR DESCRIPTION
**Pull request by olgakahraman**
still working on css

### CODE REVIEW

File: src/hooks/useLocalStorageReviews.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { useState, useEffect } from 'react';
+
+export default function useLocalStorageReviews(key, initialValue) {
```
Comment (ElijahMSmith) [inline src/hooks/useLocalStorageReviews.js]: This isn't really specific to reviews, we could pass any data in here. This is a helpful utility that should be named something generic, like `useStateWithLocalStoragePersistence` (or something shorter that captures the same idea), and we should rename `review` to `data` (also feel free to do something different here).

File: src/App.jsx [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
-						path="/find-book"
-						element={<FindABook />}
-					/>
-					<Route
-						path="/"
-						element={<Home booksData={booksData} />}
-					/>
-				</Routes>
-			</BrowserRouter>
+				<div className="container">
```
Comment (ElijahMSmith) [inline src/App.jsx:51]: Is there a reason why we need this container div?

**Review by ElijahMSmith — COMMENTED** [review]
The code here looks pretty OK. Since you haven't inserted any of the BookTalk components in any of the pages, it's hard to know what this actually looks like.

Since you are still working on CSS, I am going to hold off on hitting "approve" for now. Send us another message in the frontend channel when you are ready for someone to look again.

**Review by betymesfin — APPROVED** [review]
Great job !!! olga

**Pull request merged**